src/github/pullRequestManager.ts

```diff
@@ -180,14 +180,11 @@ export class PullRequestManager implements vscode.Disposable {
 	}
 
 	// Check if authenticated and show a prompt if not, but don't block on user's response
-	private async showLoginPrompt(): Promise<void> {
-		this._credentialStore.hasOctokit().then(authd => {
-			if (!authd) {
-				this._credentialStore.loginWithConfirmation();
-			}
-		});
-
-		return Promise.resolve();
+	private showLoginPrompt(): void {
+		const authd = this._credentialStore.isAuthenticated();
+		if (!authd) {
+			this._credentialStore.showSignInNotification();

```

Comment:
No longer wating for this?

Reply:
Right, `isAuthenticated` is now syncronous, and we previously didn't want to block on this notification